docs/security/host-certs.md

```diff
@@ -125,49 +125,56 @@ generated above.
         root@host # cp %RED%<PATH TO KEY>%ENDCOLOR% /etc/grid-security/hostkey.pem
         root@host # chmod 400 /etc/grid-security/hostkey.pem
 
-### Request certificates in bulk using osg-incommon-cert-request
+### Requesting certificates as a registration authority
+
+If you are a Registration Authority for your institution, skip ahead to [this section](#osg-incommon-cert-request).
+If you are not already a Registration Authority (RA) for your institution, you must request to be made one:
 
 1. Find your institution-specific InCommon contact
-   (e.g. [UW-Madison InCommon contact](https://it.wisc.edu/about/office-of-the-cio/cybersecurity/security-tools-software/server-certificates/)),  
-1. Ask for a Department Registration Authority user with SSL auto approve enabled. 
-1. Ask for an email invitation for a Client Certificate. Download your InCommon user certificate and split the .p12 file into your private key and your certificate:
-        
-        Extracting the private key from the .p12 file
-        :::console
-        user@host $ openssl pkcs12 -in incommon_file.p12 -nocerts -out ~/path_to_dir/incommon_user_key.pem
-        
-        Extracting the certificate from the .p12 file
-        :::console
-        user@host $ openssl pkcs12 -in incommon_file.p12 -nokeys -out ~/path_to_dir/incommon_user_cert.pem
+   (e.g. [UW-Madison InCommon contact](https://it.wisc.edu/about/office-of-the-cio/cybersecurity/security-tools-software/server-certificates/)),
+1. Request a Department Registration Authority user with SSL auto-approve enabled and client certificate:
```

Comment:
Request a Department Registration Authority user with SSL auto-approve enabled and a client certificate: